samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -584,6 +584,57 @@ static void gen_move_set_unack(struct bt_mesh_model *model,
 			       struct bt_mesh_msg_ctx *ctx,
 			       struct net_buf_simple *buf)
 {
+	u8_t tid;
+	s16_t tmp16, delta;
+	s64_t now;
+	struct net_buf_simple *msg = model->pub->msg;
+	struct generic_level_state *state = model->user_data;
+
+	delta = (s16_t) net_buf_simple_pull_le16(buf);
+	tid = net_buf_simple_pull_u8(buf);
+
+	now = k_uptime_get();
+	if (state->last_tid == tid && state->last_tx_addr == ctx->addr &&
+	    (now - state->last_msg_timestamp <= 6000)) {
+		return;
+	}
+
+	state->last_tid = tid;
+	state->last_tx_addr = ctx->addr;
+	state->last_msg_timestamp = now;
+
+	tmp16 = state->level + delta;
+	if (tmp16 < -32768) {
+		tmp16 = -32768;
```

Comment:
Is this really needed? Isn't -32768 the minimum possible value a 16-bit signed integer can hold, i.e. INT16_MIN?

Reply:
done !!

Reply:
As of now I've removed content under gen_move_set/gen_move_set_unack. Since for that transition time based support is necessary.

Reply:
Revised content under ge n_move_set/ge n_move_set_unack. So mentioned issue has resolved !!